README.md

```diff
@@ -294,15 +294,20 @@ to your local platform and then build OpenJDK and place the result in
 `/home/openjdk/target/MyOpenJDK11.tar.gz`.
 
 # Metadata
-**This is still in alpha do not rely on this yet**
 Alongside the built assets a metadata file will be created with info about the build. This will be a JSON document of the form:
 
 ```
 {
-    "WARNING": "THIS METADATA FILE IS STILL IN ALPHA DO NOT USE ME",
+    "vendor": "AdoptOpenJDK",
     "os": "mac",
     "arch": "x64",
     "variant": "openj9",
+    "jvm_version": {
+                    "major": "0",
```

Comment:
Whitespace - massive tabs?

Reply:
Thanks for spotting, will push along with other fixes.